### PR TITLE
chore: improve logging for 7.0.x

### PR DESCRIPTION
**Pull request by pgaref**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java [hunk 1]
```diff
@@ -65,8 +65,14 @@ public T getStatement() {
    * needs unmasked statement text, please use {@code getUnMaskedStatementText}
    * @return Masked statement text
    */
-  public String getStatementText() {
-    return statement.getStatementText();
+  /**
```
Comment (vcrfxia) [inline ksqldb-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java:68]: This javadoc is duplicated.

**Review by vcrfxia — APPROVED** [review]
Thanks @pgaref -- LGTM with a minor comment (nit) inline.

File: ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java [hunk 2]
```diff
@@ -131,19 +132,19 @@ static EngineExecutor create(
 
   ExecuteResult execute(final KsqlPlan plan) {
     final Optional<QueryPlan> queryPlan = plan.getQueryPlan();
-
+    final String maskedStatement = QueryMask.getMaskedStatement(plan.getStatementText());
```
Comment (vcrfxia) [inline ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java:135]: Humm... I just noticed that this change is actually redundant. Assuming we've propagated the masking through properly everywhere else, then this statement text should already be masked.

Comment (pgaref) [reply ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java:135]: Thanks! Let me look into that

Comment (pgaref) [reply ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java:135]: Planning to add the types in the master and keep these branches as clean as possible. Would that make sense?

Comment (vcrfxia) [reply ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java:135]: Makes sense. I don't mind the redundancy on these older branches as long as it's cleaned up in master, but if you want to remove the redundancy I am quite confident that it is safe (a second pair of eyes never hurts though :) )

**Pull request merged**